<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameWindow</class>
 <widget class="QMainWindow" name="GameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1605</width>
    <height>970</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1605</width>
    <height>970</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1605</width>
    <height>970</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Easy Cheese</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QScrollArea" name="gameArea">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>761</width>
      <height>841</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>759</width>
       <height>839</height>
      </rect>
     </property>
     <widget class="QLabel" name="levelLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>690</y>
        <width>341</width>
        <height>151</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="welcomeLabel">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>710</y>
        <width>461</width>
        <height>111</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="robotLabel">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>640</y>
      <width>121</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="cheese_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>680</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>90</y>
      <width>771</width>
      <height>731</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="mainLayout">
     <property name="spacing">
      <number>6</number>
     </property>
    </layout>
   </widget>
   <widget class="QPushButton" name="resetBegin">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>830</y>
      <width>20</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Reset Begin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connectButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>830</y>
      <width>20</width>
      <height>21</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 9pt &quot;Microsoft YaHei UI&quot;;</string>
    </property>
    <property name="text">
     <string>Connect Blocks</string>
    </property>
   </widget>
   <widget class="QPushButton" name="getOutput">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>820</y>
      <width>771</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 225, 127);
font: 700 9pt &quot;Microsoft YaHei UI&quot;;</string>
    </property>
    <property name="text">
     <string>Run Program!</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>1581</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="blockChoiceBar">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QGroupBox" name="conditionGroupBox">
       <property name="title">
        <string>Condition</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="notButton">
            <property name="text">
             <string>Not</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="facingPitButton">
            <property name="text">
             <string>Facing Pit</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="facingCheese">
            <property name="text">
             <string>FacingCheese</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="facingBlock">
            <property name="text">
             <string>Facing Block</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="facingWallButton">
            <property name="text">
             <string>Facing Wall</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="conditionalGroupBox">
       <property name="title">
        <string>Conditional Statement</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="ifButton">
            <property name="text">
             <string>if</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="endIfButton">
            <property name="text">
             <string>End If</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="whileButton">
            <property name="text">
             <string>While</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="endWhileButton">
            <property name="text">
             <string>End While</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="generalGroupBox">
       <property name="title">
        <string>General</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="turnLeftButton">
            <property name="text">
             <string>Turn Left</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="turnRightButton">
            <property name="text">
             <string>Turn Right</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="moveForward">
            <property name="text">
             <string>Move Forward</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="eatCheeseButton">
            <property name="text">
             <string>Eat Cheese</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>860</y>
      <width>771</width>
      <height>71</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:6pt;&quot;&gt;How to add a block to the program: Click a button at the top, then double click inside of the editor area to place the block&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:6pt;&quot;&gt;How to move a block: Click on a block and drag it around&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:6pt;&quot;&gt;How to connect a block to the program: Hold down space, then click the first block, drag and release on the second block, you will see an arrow designating that the blocks have been connected&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:6pt;&quot;&gt;How to delete a block: Click a block, then click the delete button on your keyboard&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:6pt;&quot;&gt;How to move multiple blocks: Click on an empty section of the program, then drag to highlight all the block you want to move, let go, then click on one of the block and drag to move the whole selection&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QSlider" name="speedSlider">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>360</y>
      <width>21</width>
      <height>160</height>
     </rect>
    </property>
    <property name="minimum">
     <number>100</number>
    </property>
    <property name="maximum">
     <number>1500</number>
    </property>
    <property name="singleStep">
     <number>100</number>
    </property>
    <property name="pageStep">
     <number>1000</number>
    </property>
    <property name="value">
     <number>1000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::NoTicks</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>340</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Speed</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1605</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
